update.py

```diff
@@ -756,22 +755,15 @@ def copy_static_html_sites(site, destdir):
 def check_imports():
     '''check key imports work'''
     import pkg_resources
-    requires = ["sphinx_rtd_theme>=0.1.8", "sphinxcontrib.vimeo>=0.0.1"]
+    # package names to check the versions of. Note that these can be different than the string used to import the package
+    requires = ["sphinx_rtd_theme>=0.1.8", "sphinxcontrib.youtube>=1.2.0", "sphinx==5.1.1", "docutils==0.16"]
```

Comment:
@TunaLobster the youtube check is making it impossible for me to run the script: 
![image](https://user-images.githubusercontent.com/36970042/190242126-0aeca662-ce6b-4673-9eeb-a38492ed50fd.png)

I think it might be an Ubuntu22 thing. I have made sure I have all the required packages. Any ideas what this might be? Currently just removing that check makes the script work like normal

Reply:
You will have to force install sphinxcontrib.youtube from the AP repo. `python3 -m pip install --user --force --upgrade git+https://github.com/ArduPilot/sphinxcontrib-youtube.git` Then rerun Sphinxsetup.sh to get everything else set to the correct versions.

Reply:
@TunaLobster worked like magic. Thank you!!

Reply:
I am trying to figure out a way to avoid the `--force` option. I think one way is to bump the version in the AP repo by a lot. I think that might be the least intrusive way to do it. I can't seem to find examples of telling pip to install forked packages from git over pypi packages that are already installed.